config/routes.rb

```diff
@@ -199,18 +196,18 @@
   end
 
   # Help
-  get '/help/:group', :to => 'questions#index', :as => :help
-  get '/help' => redirect('/help/basics'), :as => :faq
+  get '/help/:group' => 'questions#index', as: :help
+  get '/help' => redirect('/help/basics'), as: :faq
 
   get '/metrics' => 'metrics#overview'
 
   # Guides
-  get '/guides/:group', :to => 'guides#index', :as => :guides
-  get '/guides', :to => 'guides#index'
+  get '/guides/:group' => 'guides#index', as: :guides
+  get '/guides' => 'guides#index'
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.